src/perfEvents.h

```diff
@@ -39,6 +41,10 @@ class PerfEvents : public Engine {
     static void installSignalHandler();
     static void signalHandler(int signo, siginfo_t* siginfo, void* ucontext);
 
+    static void setActive(bool value) {
+        __atomic_store_n(&_is_active, value, __ATOMIC_SEQ_CST);
```

Comment:
```
src/perfEvents.h:45: error: ‘__ATOMIC_SEQ_CST’ was not declared in this scope
src/perfEvents.h:45: error: ‘__atomic_store_n’ was not declared in this scope
```

Reply:
gcc 4.6.4
And it is not even the oldest one - I know a customer who runs async-profiler on Linux 2.6.39 with ten years old glibc :) Enterprise IT is tough (sigh)... so I'd prefer to keep requirements and dependencies as low as possible.

Reply:
Oh...
OK, I will think about it.
Is it possible to downgrade gcc to target (4.6.4) version on CI? Travis use `g++ (Ubuntu 4.8.4-2ubuntu1~14.04.3) 4.8.4` and it give us a false confidence when check is passed.

Reply:
I switched travis configuration to `precise` dist that comes with gcc 4.6.3.

Reply:
Thanks.
One more question: can we compile with `-std=gnu++0x` and use features listed as available here: https://gcc.gnu.org/gcc-4.6/cxx0x_status.html ?

Reply:
Why? I don't see what problem this will solve, but this may definitely introduce compatibility issues. Even if you have `__atomic` intrinsics, `onThreadStart/onThreadEnd` functions are still not thread-safe.